smart-contracts/published-npm-modules/V7/ILockKeyCanacelHookV7.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.5.0;
+pragma solidity 0.5.17;
```

Comment:
Please keep the interfaces version agnostic. We don't want to block a 3rd party from using 0.5.18 when it comes out

Reply:
dang , I missed that. will fix!

Reply:
done.